src/plugins/notebook/components/SectionComponent.vue

```diff
@@ -116,16 +116,32 @@ export default {
 
             this.$emit('selectSection', id);
         },
-        updateName(event) {
-            const target = event.target;
-            target.classList.remove('c-input-inline');
-            const name = target.textContent.trim();
+        renameSection(target) {
+            if (!target) {
+                return;
+            }
+
+            const name = target.textContent = target.textContent ? target.textContent.trim().toString() : `Unnamed ${this.sectionTitle}`;
```

Comment:
```suggestion
            const name  = target.textContent ? target.textContent.trim() : `Unnamed ${this.sectionTitle}`;
```

Reply:
@ozyx, I'll remove `.toString()` function however please note that removing the `target.textContent` assignment with the default value in the case of removing the entire text will cause the header will contain the default value and the component will include the empty string. Is it the expectation?

<img width="635" alt="image" src="https://user-images.githubusercontent.com/4381303/179728540-5194005c-c0f6-4962-882e-f997c2bb5f1c.png">

Reply:
Ah, good catch. I initially thought this was a typo. In that case, can we separate out the `target.textContent` assignment to make it more clear what's happening here?

Reply:
I have done some changes in this part of the codes for the Selection and Page components